### ISSUE TITLE
[androiddebugbridge] start-package failing for apps that require an Activity

### ISSUE THREAD

**Issue by andiwirs** [openhab/openhab-addons#11551]
## Expected Behavior
Sending a string containing an App Activity in the form of `de.swr.avp.ard.tv/.TvActivity` or `de.swr.avp.ard.tv/de.swr.avp.ard.tv.TvActivity` should start the app.

## Current Behavior
Many of the apps on my Android 9 TV require the start Activity to be defined.

If i do that the following appears in the log:
`2021-11-10 05:35:05.089 [WARN ] [ge.internal.AndroidDebugBridgeDevice] - de.swr.avp.ard.tv/.TvActivity is not a valid package name`

If i leave out the activity the binding crashes and the thing goes offline:
```
2021-11-10 05:37:54.884 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'St_Tv_StartApp_Lr' changed from com.google.android.youtube.tv to com.sony.dtv.tvx
2021-11-10 05:37:56.977 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'androiddebugbridge:android:sony_tv' changed from ONLINE to OFFLINE (COMMUNICATION_ERROR): Unable to open package
```
The Youtube app for example does not need an activity there it works and the app starts:
`2021-11-10 05:36:57.582 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'St_Tv_RunningApp_Lr' changed from com.google.android.tvlauncher to com.google.android.youtube.tv`

If i issue the commands from this [link](https://stackoverflow.com/questions/4567904/how-to-start-an-application-using-android-adb-tools) with the Android platform tools (including the activites) starting all my apps works fine. So i assume this is a bug.

## Possible Solution
Maybe a long shot, but does the binding check the string for characters (in this case the '/' dividing package name and activity name) and consider this as not allowed?

## Steps to Reproduce (for Bugs)
See above.

## Context
I'd like to be able to start Apps on my TV from OH3.

## Your Environment
openHAB 3.2 M3 on NanoPi M4 Ubuntu Bionic Beaver
androiddebugbridge-Binding
Sony Bravia Android 9 TV

**Comment by andiwirs** [comment]
@GiviMAD maybe? Thanks in advance :)

**Comment by GiviMAD** [comment]
Right now the binding is using the monkey command to start the app to avoid the need of the activity name. It can use 'am' command when it detects the '/' character in the input so both will work.

**Comment by andiwirs** [comment]
Thanks for your quick answer. I manually tried the monkey command with the args i found in the source code but this happens:

```
C:\Android\platform-tools>adb shell monkey -p de.swr.avp.ard.tv --pct-syskeys 0 -v 1
  bash arg: -p
  bash arg: de.swr.avp.ard.tv
  bash arg: --pct-syskeys
  bash arg: 0
  bash arg: -v
  bash arg: 1
args: [-p, de.swr.avp.ard.tv, --pct-syskeys, 0, -v, 1]
 arg: "-p"
 arg: "de.swr.avp.ard.tv"
 arg: "--pct-syskeys"
 arg: "0"
 arg: "-v"
 arg: "1"
data="de.swr.avp.ard.tv"
arg="--pct-syskeys" mCurArgData="null" mNextArg=3 argwas="--pct-syskeys" nextarg="0"
data="0"
:Monkey: seed=1637035535197 count=1
:AllowPackage: de.swr.avp.ard.tv
:IncludeCategory: android.intent.category.LAUNCHER
:IncludeCategory: android.intent.category.MONKEY
** No activities found to run, monkey aborted.
```

Are you sure that startPackage() is looking for a "/" and then using "am" command?
I can only find the usage of "am" in stopPackage() and "contains("/")" in getCurrentPackage().

**Comment by GiviMAD** [comment]
"Are you sure that startPackage() is looking for a "/" and then using "am" command?" No, that was just a posible solution I was suggesting, sorry for the misunderstand.

**Comment by andiwirs** [comment]
Sounds great. Are you able to implement this? I'll be happy to test :)

Just for information: Another thing i noticed is that the binding doc does not seem to be up to date. 
In the [channels table](https://next.openhab.org/addons/bindings/androiddebugbridge/#channels) are only 11 items but there are actually 13 channels in my ADB Thing .

**Comment by GiviMAD** [comment]
It's included in this pr #11692

**Comment by andiwirs** [comment]
Thanks a lot :) Hopefully it still makes it into the OH 3.2 release.
